lib/lotus/root.rb

```diff
@@ -0,0 +1,7 @@
+require 'pathname'
+
+module Lotus
+  def self.root
+    Lotus::Configuration.new.root
```

Comment:
I don't the above implementation is correct though. What you did above is to retrieve the root value of an app (aka slice), what we wanted here is the root level of the container ( the directory that `config.ru` reside )

What I'd like to suggest is to look into `Lotus::Loader`, there you can work out that we should introduce a root setter for `configuration` perhaps? Let me know if you need any assistant on this ticket.